_posts/2018-01-01.md

```diff
@@ -56,6 +56,8 @@ The [last newsletter](https://neovim.io/news/2017/12/) ** INTRO HERE **
   
  - Man improvements, uses lua for speedup (**@keidax**, [#7623](https://github.com/neovim/neovim/pull/7623))
  
+ - Output from `system()` and `:!` will now be rendered nicely (**@bfredl**, [#7844](https://github.com/neovim/neovim/issues/7844))
```

Comment:
"nicely" is subjective, and it is still not always nice (colors are not supported). The main change is that it supports multibyte chars, and also doesn't crash on binary data.

Reply:
Sooooo  .... "more correctly"? "Nicer"? "Better"?

Reply:
I think listing the specific items that @bfredl did, is good enough.

Reply:
Ok, I just took it verbatim, polishing can be done when the next newsletter is due and we look for consistent wording and tone :)